### PR TITLE
RECIPE JESC for MT

### PR DESCRIPTION
**Pull request by Fhrozen**
In replace of #1249

The training in pytorch is not quite good (?)
acc
bleu

```
BLEU = 0.00, 1.5/3.1/0.0/0.0 (BP=0.990, ratio=0.990, hyp_len=2032, ref_len=2052)
```

Training in chainer have a slightly better acc but did not finish to train for 20 epochs (only 10) so it can be even better.
acc

(Forgot to implement Bleu graph in chainer).

### CODE REVIEW

**Comment by sw005320** [comment]
Can you double-check the BLEU scoring part? The accuracy looks OK.

**Comment by sw005320** [comment]
@Fhrozen, can you report the progress?

**Comment by Fhrozen** [comment]
I manage to obtain a BLEU score,
```
write a case-insensitive BLEU result in exp/train.en_lc.rm_lc.rm_pytorch_train/decode_test.jp_decode/result.lc.txt
BLEU = 13.39, 43.3/25.4/16.9/10.9 (BP=0.631, ratio=0.685, hyp_len=20767, ref_len=30338)
```

I will modify the code and move from score_bleu to local. I will be also uploading the pre-trained model.

File: egs/jesc/mt1/local/score_bleu.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```
Comment (sw005320) [inline egs/jesc/mt1/local/score_bleu.sh]: Can you add some comments on how you change from the original `score_bleu.sh`? This would help the other people to work on multi-byte chars.

Comment (Fhrozen) [reply egs/jesc/mt1/local/score_bleu.sh]: Comment added. Could you please check it?

File: egs/jesc/mt1/local/score_bleu.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+#     filt.py -v $nlsyms ${dir}/ref.wrd.trn.detok.tmp > ${dir}/ref.wrd.trn.detok
+#     filt.py -v $nlsyms ${dir}/hyp.wrd.trn.detok.tmp > ${dir}/hyp.wrd.trn.detok
+#     filt.py -v $nlsyms ${dir}/src.wrd.trn.detok.tmp > ${dir}/src.wrd.trn.detok
+# fi
+# if [ -n "${filter}" ]; then
+#     sed -i.bak3 -f ${filter} ${dir}/hyp.wrd.trn.detok
+#     sed -i.bak3 -f ${filter} ${dir}/ref.wrd.trn.detok
+#     sed -i.bak3 -f ${filter} ${dir}/src.wrd.trn.detok
+# fi
+# NOTE: this must be performed after detokenization so that punctuation marks are not removed
```
Comment (sw005320) [inline egs/jesc/mt1/local/score_bleu.sh]: You can remove all these lines.

**Comment by sw005320** [comment]
Do you want to merge it to v.0.7.0? The main change of this PR is the recipe part, and v.0.6.1 is fine for me. Also, f if you fix the test, I'll merge it.

**Comment by kan-bayashi** [comment]
Is this PR ready to merge?

**Comment by Fhrozen** [comment]
I just merged with master, so it should be fine once the test are done.

**Comment by sw005320** [comment]
Thanks a lot!!!
